### PR TITLE
Add views

### PR DESCRIPTION
**Pull request by VitorGuedesMadeira**
### Blog app - Views
[Project Requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md)
**_Learning objectives:_**
✔️ Use preprocessed HTML file with embedded Ruby code.
✔️ Use layouts and templates for shared content.

**_author:_** [Vitor Guedes Madeira](https://github.com/VitorGuedesMadeira)
tumblr_mjzccxFrE71rmbv5ro1_500

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,40 @@
...
+      </div>
+
+      <% @user.posts.each do |post| %>
+        <div class="single-user-single-post">
+          <h3>Post <%= post.id %></h3>
+          <p><%= post.text %></p>
+          <span>Comments: <%= post.comments_counter %></span>
+          <span>Likes: <%= post.likes_counter %></span>
+        </div>
+      <% end %>
```
Comment (AnselemOdims) [inline app/views/users/show.html.erb]: - [ ] Please kindly ensure that the user only sees their recent posts. You can do this by looping through using the recent post method created in the model

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1 +1,37 @@
...
+        <div class="single-user-single-post">
+          <h3>Post <%= post.id %></h3>
+          <p><%= post.text %></p>
+          <span>Comments: <%= post.comments_counter %></span>
+          <span>Likes: <%= post.likes_counter %></span>
+        </div>
+        <div class="single-user-single-post-comments">
+          <% post.comments.each do |comment| %>
+            <div class="single-user-single-post-single-comment">
+              <p><strong><%= comment.user.name %>:</strong></p>
```
Comment (AnselemOdims) [inline app/views/posts/index.html.erb]: - [ ] Same as the above review please ensure that this displays the recent comments.

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @VitorGuedesMadeira,
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: app/views/users/show.html.erb (hunk 1 above)
Comment (VitorGuedesMadeira) [reply app/views/users/show.html.erb]: Done! ✔️

File: app/views/posts/index.html.erb (hunk 2 above)
Comment (VitorGuedesMadeira) [reply app/views/posts/index.html.erb]: Done! ✔️

**Review by t-yanick — APPROVED** [review]
Hi @VitorGuedesMadeira, 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**